packages/webdriver-utils/src/metadata/mobileMetaData.js

```diff
@@ -43,6 +43,12 @@ export default class MobileMetaData {
     return { width, height };
   }
 
+  async screenResolution() {
```

Comment:
Ideally we should move this to parent class ( currently there isn't one currently)

Reply:
Currently we don't have it we can create in upcoming times though.